services/movie_session.py

```diff
@@ -0,0 +1,55 @@
+from django.db.models import QuerySet
+from datetime import datetime
+
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(
+        session_date: str | None = None
+) -> QuerySet[MovieSession]:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        movie_date = datetime.strptime(session_date, "%Y-%m-%d")
+        queryset = queryset.filter(
+            show_time__year=movie_date.year,
+            show_time__month=movie_date.month,
+            show_time__day=movie_date.day
+        )
+    return queryset
```

Comment:
```suggestion
    if session_date:
        return MovieSession.objects.filter(show_time__date=session_date)
    return MovieSession.objects.all()

```

it can be done more easily)